src/validation/validators/matchField/MatchFieldValidation.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { MatchFieldError } from "@/validation/errors/MatchFieldError";
+import { MatchFieldValidation } from "./MatchFieldValidation";
+import faker from "faker";
+
+const ONLY_DIGITS_REGEX = /^[0-9]*$/;
+const CAPITAL_LETTERS_REGEX = /[A-Z]+/;
+
+const makeDigitsSut = (fieldName: string): MatchFieldValidation =>
+  new MatchFieldValidation(fieldName, ONLY_DIGITS_REGEX);
+
+const makeCapitalLettersSut = (
+  fieldName: string,
+  ignoreCase: boolean,
+  message?: string
+): MatchFieldValidation =>
+  new MatchFieldValidation(
+    fieldName,
+    CAPITAL_LETTERS_REGEX,
+    ignoreCase,
+    message
+  );
+
+describe("MatchFieldValidation", () => {
+  test("Should return error if valid is invalid", () => {
```

Comment:
I think it should be 'Should return error if match is invalid for Invalid Digit Regex'